vendor/api/v1/authorizenet/views.py

```diff
@@ -162,38 +165,45 @@ class AuthorizeCaptureAPI(AuthorizeNetBaseAPI):
     API endpoint to get event notifications from authorizenet when a authcaputre is created.
     If there is a subscription tied to the transaction, it will renew such subscription
     """
-
-    def post(self, *args, **kwargs):
-        logger.info(f"AuthorizeCaptureAPI post: Event webhook: {self.request.body}")
-        site = get_site_from_request(self.request)
-        logger.info(f"AuthorizeCaptureAPI post: site: {site}")
-
+    def is_request_valid(self, site, request_data):
         if not self.request.body:
```

Comment:
since `request_data` is `self.request.body`, couldnt you just do `if not request_data`?

Reply:
good catch thanks!!!1